src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -62,6 +62,37 @@ std::shared_ptr<IPassManager> BasePass::getPassManager() {
     return sharedMgr;
 }
 
+static auto DoNotSkip = [](CNNLayerPtr) {
+    return false;
+};
+
+/**
+ * @brief Perform addition of two blobs values
+ */
+static void SumBlobs(Blob::Ptr& src_blob, Blob::Ptr& dst_blob) {
+    if (!src_blob || !dst_blob || src_blob->size() != dst_blob->size()) {
+        return;
+    }
+    float* src_blob_buf = src_blob->buffer().as<float*>();
+    float* dst_blob_buf = dst_blob->buffer().as<float*>();
+    std::transform(dst_blob_buf, dst_blob_buf + dst_blob->size(), src_blob_buf, dst_blob_buf, std::plus<float>());
+}
+
+/**
+ * @brief Get next layer
+ */
+static CNNLayerPtr GetNextLayer(CNNLayerPtr layer) {
```

Comment:
I see that in gna_pass_manager 
there is already used one method to get next layer 

CNNNetCheckNextLayerSkipCertain

Is there a reason why we need second solution?

Reply:
No, changed.